src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,96 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_SIZE = 10;
+    public static final double ADD_SIE = 1.5;
```

Comment:
they are used only in this class, so should be `private`
```suggestion
    private static final int DEFAULT_SIZE = 10;
    private static final double ADD_SIE = 1.5;
```